somnio_starter/__brick__/{{project_name.snakeCase()}}/pubspec.yaml

```diff
@@ -14,30 +14,30 @@ dependencies:
     path: packages/app_config_repository
   auth_client:
     path: packages/auth_client
-  bloc: ^8.1.3
+  bloc: ^8.1.4
   equatable: ^2.0.5
   flutter:
     sdk: flutter
-  flutter_bloc: ^8.1.4
+  flutter_bloc: ^8.1.6
   flutter_localizations:
     sdk: flutter
   form_inputs:
     path: packages/form_inputs
-  go_router: ^14.2.3
+  go_router: ^14.2.9
   intl: ^0.19.0
-  {{project_name.snakeCase()}}_ui:
-    path: packages/{{project_name.snakeCase()}}_ui
-  package_info_plus: ^8.0.0
-  url_launcher: ^6.2.6
+  package_info_plus: ^8.0.2
+  temp_ui:
```

Comment:
I think having this as a brick parameter `{{project_name.snakeCase()}}_ui` is better so it won't be an error after creating it, right?